include/rocksdb/table.h

```diff
@@ -125,6 +125,16 @@ struct CacheUsageOptions {
   std::map<CacheEntryRole, CacheEntryRoleOptions> options_overrides;
 };
 
+class CompactionPrepulateBlockCachePolicy {
```

Comment:
Great idea on the callback, that also allows us to dynamically change the setting without rolling our processes!